### PR TITLE
add flag -log-only-denied

### PR DESCRIPTION
**Pull request by flixr**
To reduce unnecessary logs but not be completely quiet:
* log decision in single line
* add flag -log-only-denied

### CODE REVIEW

**Review by anderseknert — APPROVED** [review]
👍 thanks!

**Comment by anderseknert** [comment]
You'll need to add a sign-off to your commit for the DCO check before this can be merged though.

Just:
```
git commit --amend --signoff
git commit push --force
```

**Comment by flixr** [comment]
done..

**Review by anderseknert — APPROVED** [review]

**Pull request merged**
